src/main/scala/org/broadinstitute/dsde/workbench/sam/service/PolicyEvaluatorService.scala

```diff
@@ -13,6 +14,20 @@ class PolicyEvaluatorService(
     private val resourceTypes: Map[ResourceTypeName, ResourceType],
     private val accessPolicyDAO: AccessPolicyDAO)(implicit val executionContext: ExecutionContext)
     extends LazyLogging {
+  def initPolicy(): IO[Unit] = {
+    val policyName = AccessPolicyName("admin-notifier-set-public")
+    accessPolicyDAO.createPolicy(AccessPolicy(
+      FullyQualifiedPolicyId(FullyQualifiedResourceId(SamResourceTypes.resourceTypeAdminName, ResourceId("managed-group")), policyName),
+      Set.empty,
+      WorkbenchEmail("dummy@gmail.com"), //TODO(Qi): make this valid google email domain after https://github.com/broadinstitute/sam/pull/230
+      Set.empty,
+      Set(SamResourceActions.setPublicPolicy(policyName)),
+      true
+    )).void.recover{
+      case ldape: LDAPException if ldape.getResultCode == ResultCode.ENTRY_ALREADY_EXISTS => ()
```

Comment:
I get nervous whenever we eat an exception and don't do anything.  I advocate at least a log statement.

Reply:
this particular exception just means we've created this policy before (which in most cases is true since we only need to create it once in each ldap instance). But I guess it's still worth adding a log

Reply:
Yeah, seen too many troubleshooting issues where exceptions were swallowed and we waste a lot of time finding it.  Your code might be fine now, but some day somebody modifies it and it's suddenly not OK and we have something nasty to track down.

Reply:
not disagreeing...it's updated https://github.com/broadinstitute/sam/pull/233/commits/60c3c7a60f463045251439e8591d553ba8a5a0f4